geometry_msgs/Vector3 pose
geometry_msgs/Vector3 pose_base
geometry_msgs/Vector3 pose_odom
geometry_msgs/Vector3 angle
float32 distance